### PR TITLE
add clearGlobals function and movie ticks prop

### PR DESCRIPTION
**Pull request by matthewdean**

### CODE REVIEW

File: vm-rust/src/player/movie.rs [hunk 1]
```diff
@@ -90,6 +91,7 @@ impl Movie {
       "traceLogFile" => Ok(Datum::String("".to_string())), // TODO
       "traceScript" => Ok(Datum::Int(0)), // TODO
       "movieName" => Ok(Datum::String(self.file_name.to_owned())),
+      "ticks" => Ok(Datum::Int(utils::get_elapsed_ticks(self.timer_tick_start))),
```
Comment (igorlira) [inline vm-rust/src/player/movie.rs:94]: This prop can be moved to `DirPlayer::get_movie_prop`, that would eliminate the need for a `timer_tick_start` field in the Movie itself.

**Review by igorlira — COMMENTED** [review]
Thanks for contributing! This looks great, I just left a minor comment about moving the `ticks` prop getter to the DirPlayer struct itself to prevent a field from being duplicated.

File: vm-rust/src/player/movie.rs (hunk 1 above)
Comment (matthewdean) [reply vm-rust/src/player/movie.rs]: Done.

**Review by igorlira — APPROVED** [review]
Looks great, thank you!

**Pull request merged**